src/components/PermissionsMap.js

```diff
@@ -0,0 +1,159 @@
+// permissionsMap.js
+
```

Comment:
can this file be deleted in favor of `PermissionsMap.tsx`?

Reply:
Yes - this was just testing out a proof of concept in a node environment.